### PR TITLE
Getting started

### PR DESCRIPTION
**Pull request by waderyan**
There is also a PR in vscode-website. 

Some previews. 

image

image

image

image

image

image

image

### CODE REVIEW

File: docs/introVideos/basics.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Basics and Setup of VS Code
+
+In this tutorial we walk you through the basics of Visual Studio Code. 
+
+<iframe src="https://www.youtube.com/embed/fpqy0ZkavWM?rel=0&amp;disablekb=0&amp;modestbranding=1&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+## Outline
+
+* Download and Install VS Code
```
Comment (gregvanl) [inline docs/introVideos/basics.md]: Download and install VS Code
Practice **Quick Open** and using the **Command Palette**

File: docs/introVideos/basics.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+TOCTitle: Basics and Setup
+ContentId: baf150cd-6daf-4604-87db-a7c70a6706a7
+PageTitle: Basics and Setup of VS Code
+DateApproved: 8/10/2016
+MetaDescription: Download and learn the basics of Visual Studio Code.
+---
+
+# Basics and Setup of VS Code
+
+In this tutorial we walk you through the basics of Visual Studio Code.
```
Comment (gregvanl) [inline docs/introVideos/basics.md]: In this tutorial, we walk...

File: docs/introVideos/debugging.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+# Debugging in VS Code
+
+Debugging is a core feature of VS Code. In this tutorial we will show you how to configure and use the debugging basics. We will walk you through debugging a Node.js application. 
+
+<iframe src="https://www.youtube.com/embed/hvPuPi8iG_w?rel=0&amp;disablekb=0&amp;modestbranding=1&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
+
+## Outline
+
+* Configure debugging by creating a launch.json file
```
Comment (gregvanl) [inline docs/introVideos/debugging.md]: ...creating a `launch.json` file

File: docs/introVideos/debugging.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Set break points
+* Inspect your program's state and variables
+* Step through your code
```
Comment (gregvanl) [inline docs/introVideos/debugging.md]: Step through your source code

File: docs/introVideos/debugging.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* Use the debug view and console, including variables, call stack information, and breakpoints.
```
Comment (gregvanl) [inline docs/introVideos/debugging.md]: Use the Debug View and Console, including...

File: docs/introVideos/debugging.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+* Configure debugging by creating a launch.json file
+* Set break points
+* Inspect your program's state and variables
+* Step through your code
+* Use the debug view and console, including variables, call stack information, and breakpoints. 
+
+## Learn More
+
+* [Debugging](/docs/editor/debugging) - Official documentation for VS Code debugging.
+* [Integrated Terminal](/docs/editor/integrated-terminal) - Use your terminal inside VS Code.
```
Comment (gregvanl) [inline docs/introVideos/debugging.md]: - Use the integrated terminal inside VS Code.

File: docs/introVideos/overview.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+TOCTitle: Overview
+ContentId: f29747b9-0f4f-4f62-b0a7-037ffc49d972
+PageTitle: Intro Videos Overview
+DateApproved: 8/10/2016
+MetaDescription: Overview of VS Code's Introductory Videos. 
+---
+
+# Introductory Videos
+
+Start your journey with VS Code with this sequenced course of introductory videos. These videos are designed to give you a quick overview of various topics. Because VS Code comes with JavaScript and TypeScript support out of the box, each video uses JavaScript as an example.
```
Comment (gregvanl) [inline docs/introVideos/overview.md]: Start your journey with Visual Studio Code with this set of introductory videos. The videos are designed to give you a quick overview of the various features of VS Code. VS Code comes with JavaScript and TypeScript support out of the box so the videos use JavaScript examples.

File: docs/introVideos/basics.md (hunk 2 above)
Comment (waderyan) [inline docs/introVideos/basics.md]: done.

File: docs/introVideos/basics.md (hunk 1 above)
Comment (waderyan) [inline docs/introVideos/basics.md]: done.

File: docs/introVideos/debugging.md (hunk 3 above)
Comment (waderyan) [inline docs/introVideos/debugging.md]: done

File: docs/introVideos/debugging.md (hunk 4 above)
Comment (waderyan) [inline docs/introVideos/debugging.md]: done.

File: docs/introVideos/debugging.md (hunk 5 above)
Comment (waderyan) [inline docs/introVideos/debugging.md]: done.

File: docs/introVideos/debugging.md (hunk 6 above)
Comment (waderyan) [inline docs/introVideos/debugging.md]: done.

File: docs/introVideos/overview.md (hunk 7 above)
Comment (waderyan) [inline docs/introVideos/overview.md]: done.

**Pull request merged**